website/styles.css

```
@@ -0,0 +1,252 @@
+:root {
+  --bg-color: #000000;
+  --hero-bg: linear-gradient(180deg, #5c632e 0%, #000000 100%); /* Olive gradient from screenshot */
+  --accent-color: #ff453a;
+  --text-primary: #ffffff;
+  --text-secondary: rgba(255, 255, 255, 0.6);
+  --card-bg: #1c1c1e;
+  --pill-bg: rgba(255, 255, 255, 0.1);
+  --font-main: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)